java/client/src/main/java/glide/api/commands/ListCommands.java

```diff
@@ -0,0 +1,124 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * List Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=list">List Commands</a>
+ */
+public interface ListCommands {
+    /**
+     * Inserts all the specified values at the head of the list stored at <code>key</code>. <code>
+     * elements</code> are inserted one after the other to the head of the list, from the leftmost
+     * element to the rightmost element. If <code>key</code> does not exist, it is created as an empty
+     * list before performing the push operations.
+     *
+     * @see <a href="https://redis.io/commands/lpush/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @param elements The elements to insert at the head of the list stored at <code>key</code>.
+     * @return The length of the list after the push operations.<br>
```

Comment:
```suggestion
     * @return The length of the <code>list</code> after the push operations.<br>
```

Reply:
Please apply this to other commands too

Reply:
Only highlight if it's Java-related classes or code.  The list here is a Redis list, and we haven't been emphasizing Redis types.